### ISSUE TITLE
CRAN check error

### ISSUE THREAD

**Issue by gagolews** [ThinkR-open/gitdown#14]
Hello! I'm submitting stringi 1.6.2 to CRAN now and have been notified about the auto-check problems.

```
Package: gitdown
Check: tests
New result: ERROR
    Running ‘testthat.R’ [5s/5s]
  Running the tests in ‘tests/testthat.R’ failed.
  Complete output:
    > library(testthat)
    > library(gitdown)
    > 
    > test_check("gitdown")
       inline R code fragments
    
    /usr/bin/pandoc +RTS -K512m -RTS index.knit.md --to html4 --from markdown+autolink_bare_uris+tex_math_single_backslash --output index.html --lua-filter /home/hornik/lib/R/Library/4.2/x86_64-linux-gnu/bookdown/rmarkdown/lua/custom-environment.lua --lua-filter /home/hornik/lib/R/Library/4.2/x86_64-linux-gnu/rmarkdown/rmarkdown/lua/pagebreak.lua --lua-filter /home/hornik/lib/R/Library/4.2/x86_64-linux-gnu/rmarkdown/rmarkdown/lua/latex-div.lua --metadata-file /home/hornik/tmp/scratch/RtmpCWdyBB/file7d733b313979 --wrap preserve --standalone --section-divs --table-of-contents --toc-depth 3 --template /home/hornik/lib/R/Library/4.2/x86_64-linux-gnu/bookdown/templates/gitbook.html --highlight-style pygments --number-sections --css style.css --include-in-header /home/hornik/tmp/scratch/RtmpCWdyBB/rmarkdown-str7d7359f98eda.html --mathjax 
       inline R code fragments
    
    /usr/bin/pandoc +RTS -K512m -RTS index.knit.md --to html4 --from markdown+autolink_bare_uris+tex_math_single_backslash --output index.html --lua-filter /home/hornik/lib/R/Library/4.2/x86_64-linux-gnu/bookdown/rmarkdown/lua/custom-environment.lua --lua-filter /home/hornik/lib/R/Library/4.2/x86_64-linux-gnu/rmarkdown/rmarkdown/lua/pagebreak.lua --lua-filter /home/hornik/lib/R/Library/4.2/x86_64-linux-gnu/rmarkdown/rmarkdown/lua/latex-div.lua --metadata-file /home/hornik/tmp/scratch/RtmpCWdyBB/file7d73ab9af20 --wrap preserve --standalone --section-divs --table-of-contents --toc-depth 3 --template /home/hornik/lib/R/Library/4.2/x86_64-linux-gnu/bookdown/templates/gitbook.html --highlight-style pygments --number-sections --css style.css --include-in-header /home/hornik/tmp/scratch/RtmpCWdyBB/rmarkdown-str7d732491a00f.html --mathjax 
       inline R code fragments
    
    /usr/bin/pandoc +RTS -K512m -RTS index.knit.md --to html4 --from markdown+autolink_bare_uris+tex_math_single_backslash --output index.html --lua-filter /home/hornik/lib/R/Library/4.2/x86_64-linux-gnu/bookdown/rmarkdown/lua/custom-environment.lua --lua-filter /home/hornik/lib/R/Library/4.2/x86_64-linux-gnu/rmarkdown/rmarkdown/lua/pagebreak.lua --lua-filter /home/hornik/lib/R/Library/4.2/x86_64-linux-gnu/rmarkdown/rmarkdown/lua/latex-div.lua --metadata-file /home/hornik/tmp/scratch/RtmpCWdyBB/file7d73484a95e0 --wrap preserve --standalone --section-divs --table-of-contents --toc-depth 3 --template /home/hornik/lib/R/Library/4.2/x86_64-linux-gnu/bookdown/templates/gitbook.html --highlight-style pygments --number-sections --css style.css --include-in-header /home/hornik/tmp/scratch/RtmpCWdyBB/rmarkdown-str7d736c53e814.html --mathjax 
       inline R code fragments
    
    /usr/bin/pandoc +RTS -K512m -RTS index.knit.md --to html4 --from markdown+autolink_bare_uris+tex_math_single_backslash --output index.html --lua-filter /home/hornik/lib/R/Library/4.2/x86_64-linux-gnu/bookdown/rmarkdown/lua/custom-environment.lua --lua-filter /home/hornik/lib/R/Library/4.2/x86_64-linux-gnu/rmarkdown/rmarkdown/lua/pagebreak.lua --lua-filter /home/hornik/lib/R/Library/4.2/x86_64-linux-gnu/rmarkdown/rmarkdown/lua/latex-div.lua --metadata-file /home/hornik/tmp/scratch/RtmpCWdyBB/file7d731cd346d9 --wrap preserve --standalone --section-divs --table-of-contents --toc-depth 3 --template /home/hornik/lib/R/Library/4.2/x86_64-linux-gnu/bookdown/templates/gitbook.html --highlight-style pygments --number-sections --css style.css --include-in-header /home/hornik/tmp/scratch/RtmpCWdyBB/rmarkdown-str7d731bf79f3e.html --mathjax 
    ══ Failed tests ════════════════════════════════════════════════════════════════
    ── Failure (test-get_info_files.R:13:3): get_info works ────────────────────────
    as.Date(infos$first_modif) not equal to setNames(today, "first").
    Mean relative difference: 5.329638e-05
    ── Failure (test-get_info_files.R:14:3): get_info works ────────────────────────
    as.Date(infos$last_modif) not equal to setNames(today, "last").
    Mean relative difference: 5.329638e-05
    ── Failure (test-get_info_files.R:19:3): get_info works ────────────────────────
    as.Date(infos_not_in_repo$last_modif) not equal to setNames(today, "last").
    Mean relative difference: 5.329638e-05
    
    [ FAIL 3 | WARN 0 | SKIP 0 | PASS 72 ]
    Error: Test failures
    Execution halted

```

https://cran.r-project.org/web/checks/check_results_gitdown.html lists the same issue.

I hope it is not an issue related to the update of stringi?
Weird that it's only present under `r-devel-windows-x86_64-gcc10-UCRT`...

**Comment by statnmap** [comment]
Thank you for putting this into my attention. I do not think it is a problem with {stringi}.   
I'll check this out.

**Comment by statnmap** [comment]
New fix, and updated on CRAN: 16db94c0920a678a13c0c4896f64ea6a1738492d
